mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -62,6 +63,21 @@
 event_id = {'Sync On': 5}
 
 
+def test_orig_units():
+    """Test exposure of original channel units."""
+    with pytest.warns(RuntimeWarning, match='will be dropped'):
```

Comment:
In this case since the warning is unrelated and we are not testing any other warning I would use pytest's fixture `reacwarn`.

```py
def test_orig_units(recwarn):
    orig_units = read_raw_brainvision(vhdr_path)._orig_units
	assert len(orig_units) == 32
￼    assert orig_units['FP1'] == u'µV'
	...
```
￼

Reply:
I don't know about reacwarn, could you elaborate or share a link? I can't find something on Google

Reply:
Sorry, it was a typo. But the snipped is correct. Just add `recwarn` as a test parameter, and the whole thing works.

```diff
- def test_orig_units():
+ def test_orig_units(recwarn):
```
```diff
-    with pytest.warns(RuntimeWarning, match='will be dropped'):
-        raw = read_raw_brainvision(vhdr_path)
-    orig_units = raw._orig_units
+    orig_units = read_raw_brainvision(vhdr_path)._orig_units
```